dbt_platform_helper/default-extensions.yml

```diff
@@ -17,3 +17,10 @@ monitoring:
   environments:
     "*":
       enable_ops_center: false
+
+prometheus-prod:
```

Comment:
Why is it called prod? Is there a non-prod policy coming in a later PR? Is there anything to stop this extension being deployed to a non-prod env?

Reply:
Good question.  There's a prod and non prod grafana stack.  We're adding configuration for services to use prometheus prod by default, and if for some reason they want to push metrics to prometheus dev, they can add an explicit `prometheus-policy` extension with the dev assume role arn.

Reply:
I wonder if the demodjango toolspr env should connect up with prometheus dev, so that we can regression test the extension

Reply:
So my plan was to wire up demodjango to prod prometheus, but we can point the toolspr env at dev if this makes more sense.

Reply:
I've dropped the `-prod` bit

Reply:
If it's default for all services, then I think we should add a check to demodjango landing page that it's wired up correctly

Reply:
Hmm, how would that work?

Reply:
Maybe it's not possible. I thought maybe it could hit a metrics endpoint or similar

Reply:
Hmm, it's going to be tricky.  There is a metrics endpoint on the ADOT container, maybe demodjango could reverse scrape that and figure out if it is scraping metrics from the metadata endpoint.  

I'm going to create another ticket to look at this, so we can get this merged and moving forward.

Reply:
Sounds sensible. Just approved now